angular-primeng-app/src/app/pipes/truncate.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { TruncatePipe } from './truncate.pipe';
+
```

Comment:
@DongreJaipal please remove this file the - `angular-primeng-app/src/app/pipes/truncate.pipe.spec.ts`